app/routes/admin/groups/components/GroupMembers.js

```diff
@@ -89,13 +89,13 @@ export const GroupMembers = ({
 );
 
 function loadData({ query, match: { params }, location }, dispatch) {
-  const showDescendants = location.search.includes('descendants=true');
+  const showDescendants = window.location.search.includes('descendants=true');
   return dispatch(fetchMemberships(params.groupId, showDescendants, query));
 }
 
 function mapStateToProps(state, props) {
   const { pathname, search } = state.router.location;
-  const showDescendants = location.search.includes('descendants=true');
+  const showDescendants = window.location.search.includes('descendants=true');
```

Comment:
This should just be `search` from line `97`. Nice that eslint caught this bug tho! :100:

Reply:
```suggestion
  const showDescendants = search.includes('descendants=true');
```

Reply:
Will add this to my amend commit to 81cb617